ajax/libs/videojs-wavesurfer/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "videojs-wavesurfer",
+  "version": "1.3.2",
+  "description": "video.js plugin that adds a navigable waveform for audio and video files.",
+  "filename": "videojs.wavesurfer.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/collab-project/videojs-wavesurfer.git"
+  },
+  "keywords": [
+    "waveform",
+    "audio",
+    "video",
+    "wavesurfer",
+    "videojs",
+    "videojs-plugin",
+    "player"
+  ],
+  "author": "Collab",
+  "license": "MIT",
+  "homepage": "http://videojs.com/",
```

Comment:
@hare1039 I think this should discuss.
We host a lot of the plugin of videojs, such as https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/videojs-swf/package.json
And the library `videojs-swf` homepage is "http://videojs.com/".